docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
+You can add data from multiple indices to a Timeline to facilitate advanced investigations.
 
-You can drag or send items of interest to Timeline to create the query you need
-to investigate an alert or event. You can add items from tables and histograms
+You can drag or send items of interest to a Timeline to create the desired query. You can add items from tables and histograms
 on the *Overview*, *Alerts*, *Hosts*, and *Network* pages, as well as from
-within a Timeline itself. Additionally, you can add a query directly in Timeline
-(click *+ Add field*).
+other Timelines. Additionally, you can add a query directly in Timeline
+by clicking *+ Add field*.
 
 [role="screenshot"]
 image::images/timeline-ui-updated.png[]
 
 Timelines are responsive, and they persist as you move through the {es-sec-app}
 collecting data. Auto-saving ensures that the results of your investigation are
-available for review by other analysts and incident response teams. To record
-and share your findings with others, attach your Timeline to a
-<<cases-overview, case>>.
+available for later review. To record and share your findings with others, attach your Timeline to a <<cases-overview, case>>.
 
-TIP: An untitled Timeline is saved as a draft. To attach a Timeline to a
+TIP: Untitled Timelines are saved as drafts. To attach a Timeline to a
 <<cases-overview, case>>, you must give it a title.
 
 In addition to Timelines, you can create and attach Timeline templates to
 <<detection-engine-overview, detection rules>>. Timeline templates allow you to
-define the source event fields used when you investigate a detections alert in
+define the source event fields used when you investigate alerts in
 Timeline. You can select whether the fields use predefined values or values
-retrieved from the alert. For more information, see <<timeline-templates-ui>>.
+retrieved from the alert. For more information, refer to <<timeline-templates-ui>>.
 
 
 [discrete]
 [[refine-timeline-results]]
 == View and refine Timeline results
 
-Using the drop-down options
-by the KQL bar, you can select whether <<det-engine-terminology, alerts>>,
-other raw events, or both are displayed in the Timeline.
+Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>, other raw events, or both are displayed in the Timeline.
 
 [discrete]
 [[conf-timeline-display]]
 == Configure Timeline event context and display
 
-Many events in Timeline are presented in easy-to-follow rendered views that
-include relevant contextual information. These views are called
-*Event Renderers*, and they can be configured via the Settings icon in the upper
-left corner of the result's pane:
+Many types of events automatically appear in preconfigured views that provide various kinds of context, called *Event Renderers*. You can see and turn them on or off via the Settings icon in the upper left corner of the results pane:
 
 [role="screenshot"]
 image::images/timeline-ui-renderer.png[]
```

Comment:
Here's the current image:
<img width="1851" alt="timeline-ui-renderer" src="https://user-images.githubusercontent.com/79124755/156790590-d879db57-4421-4dce-a3dd-02862c76188b.png">